cmd/root.go

```diff
@@ -372,6 +389,14 @@ func initConfig() {
 	if len(dataDirectory) != 0 {
 		Config.DataDirectory = dataDirectory
 	}
+
+	if allInMemory {
+		Config.AllInMemoryEnabled = allInMemory
+	}
+
+	if tableSize != 0 {
```

Comment:
Why should I care about this table size when I run cli?

Reply:
tableSize enable users to control the memory usage. in the default mode is 1GB
if we make it to a higher number, reading will be faster, compact frequency will be lower, generally means: faster read write and more memory usage,
I enable this config is to allow users to config it based on different instances they choosed

Reply:
do we even care about compaction - it's not long running database?

Reply:
L0 care about it, and the tablesize influence how many tables we have. for read a value, more tables sometimes means slower. as more bloomfilter actions